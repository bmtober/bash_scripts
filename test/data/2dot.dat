Homer Bart
Homer Lisa
Marge Bart
Marge Lisa
Lisa  Bart
