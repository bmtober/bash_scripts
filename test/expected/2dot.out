digraph {
overlap=scale;
"Homer" [label="Homer"];
"Marge" [label="Marge"];
"Lisa" [label="Lisa"];
"Bart" [label="Bart"];

"Homer" -> "Bart";
"Homer" -> "Lisa";
"Marge" -> "Bart";
"Marge" -> "Lisa";
"Lisa" -> "Bart";
}
